operations/hpe_pdu/hpe_pdu_admin_procedures.md

```diff
@@ -0,0 +1,78 @@
+## HPE PDU Admin Procedures
+* [Connecting to HPE PDU Web Interface](#connecting-to-hpe-pdu-web-interface)
+* [HPE PDU Initial Set-up](#hpe-pdu-initial-set-up)
+* [Updating HPE PDU Firmware](#updating-hpe-pdu-firmware)
+* [Changing HPE PDU User Passwords](#changing-hpe-pdu-user-passwords)
+* [Discovery of HPE PDU after upgrading CSM](#discovery-of-hpe-pdu-after-upgrading-csm)
+
+**Because of the pulling method used to process sensor data from the HPD PDU, telemetry data may take up to 6 minutes to refresh, this includes outlet status, voltage and other data from the PDU.**
+
+
+### Connecting to HPE PDU Web Interface
+You will need to know the:
+* IP address or Domain Name of m001
+* Xname of the HPE PDU
+* Root password for m001
+* Admin password for HPE PDU _(default: 12345678)_
+
+1. Use the SSH command from your local machine:\
+`> ssh -L 8443:{PDU_xname}:443 -N root@{m001_ip}`\

```

Comment:
Maybe change this to be:
````
```
linux# ssh -L 8443:{PDU_xname}:443 -N root@{m001_ip}
```
````

Reply:
This command is run on the local machine, so could be a Windows / MacOS / Linux box, so do not want to use the `linux#` prompt

Reply:
In that case, I would mentation that that the command can work on Windows, macOS, and Linux. I don't think we cover this case in the https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#command-prompt-conventions

Reply:
Updated